docs/requirements-insiders.txt

```diff
@@ -1,5 +1,5 @@
 PyYAML==6.0.2
-ruff==0.7.4
+ruff==0.8.0
```

Comment:
Do we need to update Ruff in this PR? I would prefer to avoid making unrelated changes.